### DISCUSSION TITLE
Using pre-built app

### DISCUSSION THREAD

**Discussion by bewster** [YannickRe/spfx-msgraph-peoplesearch#27] [Q&A] [answered]
Do I need to do anything special to use the version you released?

![image](https://user-images.githubusercontent.com/3236592/122435761-2d77d380-cf90-11eb-9244-7bb664894acd.png)

I thought it would be a quick way of getting started (instead of building it myself).

**Comment by bewster** [comment, marked as answer]
Sorry, my inexperience is showing.

As I checked "Make this solution available to all sites in the organization" it doesn't appear in the list.

**Reply by YannickRe** [reply]
Correct, that way the webpart will immediately show up in the webpart toolbox on each page of each site.
Have fun!
